tests/conftest.py

```diff
@@ -16,7 +18,47 @@ def app():
     with app.app_context():
         db.drop_all()
 
-
 @pytest.fixture
 def client(app):
     return app.test_client()
+
+@pytest.fixture
+def add_two_boards(app):
+    board_1 = Board(title="title_1", owner="owner_1")
```

Comment:
If we will need to assert about info on boards or cards in the tests, we could create a constants file for the test data like titles, owners, and messages then import the variables into the conftest and test files. This lets us define values in one place and reuse them, no duplicating the same string across many files or worrying about a letter being off here or there while retyping something.